### ISSUE TITLE
Migrate to newer version of Fundamentals. 

### ISSUE THREAD

**Issue by fkolar** [ngx-meta/rules#54]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[x ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
[ ] Other... Please describe:
</code></pre>

## Current behavior

Currently we are running againts old version of fundamentals and to be able to move this UI development to actual fundamentals library so it can be part of it. 

## Expected behavior
 - Update libraries, 
 - move to fundamentals git 
 - remove code from this git.

**Comment by GregOnNet** [comment]
Hi, ist fundamentals a library?

**Comment by fkolar** [comment]
yeah, its https://github.com/SAP/fundamental-ngx, and next week hopefully I will be able to to do it.
